lib/proc/proc_select.cpp

```diff
@@ -4914,6 +4914,10 @@ namespace {
       if (!grn_select_post_filter(ctx, data)) {
         goto exit;
       }
+      if (data->filter.post_filter.length == 0) {
```

Comment:
```suggestion
      if (data->filter.post_filter.length != 0) {
```

Reply:
fix: [a9d9ee6](https://github.com/groonga/groonga/pull/1754/commits/a9d9ee6fa830fbf0e44c0e00eb017efdfbc53fcb) We have to skip it when we don't use the post_filter. It's invert condition.